app/routes.py

```diff
@@ -0,0 +1,274 @@
+    if "total_inventory" not in request_body:
+        return make_response(jsonify({"details" : "Request body must include total_inventory."}), 400)
+    
+    new_video = Video(
+        title=request_body["title"],
+        release_date = request_body["release_date"],
+        total_inventory = request_body["total_inventory"],
+        )
+    db.session.add(new_video)
+    db.session.commit()
+
+    return make_response(jsonify(new_video.to_dict()),201)
+
+@customer_bp.route("", methods=["POST"])
+def create_customer():
+    request_body = request.get_json()
+
+    if "name" not in request_body:
+        return make_response(jsonify({"details" : "Request body must include name."}), 400)
+    if "postal_code" not in request_body:
+        return make_response(jsonify({"details" : "Request body must include postal_code."}), 400)
+    if "phone" not in request_body:
+        return make_response(jsonify({"details" : "Request body must include phone."}), 400)
+    
+    new_customer = Customer(
+        name=request_body["name"],
+        postal_code = request_body["postal_code"],
+        phone = request_body["phone"],
+    )
+
+    db.session.add(new_customer)
+    db.session.commit()
+
+    return make_response(jsonify(new_customer.to_dict()),201)
+
+@rental_bp.route("/check-out",methods=["POST"])
+def new_rental():
+    request_body = request.get_json()
+
+    if "video_id" not in request_body or "customer_id" not in request_body:
+        return make_response(jsonify({"details" : "Request body must include video_id and customer_id."}), 400)
+
+    validate_video_id(request_body["video_id"])
+    validate_customer_id(request_body["customer_id"])
+
+    customer = Customer.query.get(request_body["customer_id"])
+    video = Video.query.get(request_body["video_id"])
+    due_date = Rental.generate_due_date()
+
+    new_rental = Rental(
+        customer= customer,
+        video= video,
+        due_date= due_date,
+    )
+    if video.remaining_videos() < 0:
+        return make_response(jsonify({"message" : "Could not perform checkout"}), 400)
+    
+    db.session.add(new_rental)
+    db.session.commit()
+
+    return new_rental.to_dict()
+
+@rental_bp.route("/check-in",methods=["POST"])
+def rental_check_in():
+    request_body = request.get_json()
+
+    if "video_id" not in request_body or "customer_id" not in request_body:
+        return make_response(jsonify({"details" : "Request body must include video_id and customer_id."}), 400)
+
+    validate_video_id(request_body["video_id"])
+    validate_customer_id(request_body["customer_id"])
+
+    customer = Customer.query.get(request_body["customer_id"])
+    video = Video.query.get(request_body["video_id"])
+
+    if not Rental.query.filter(Rental.video_id == video.id, Rental.customer_id == customer.id).first():
+        return make_response(jsonify({"message": "No outstanding rentals for customer 1 and video 1"}), 400)
```

Comment:
You left in the hardcoded placeholders here:

```suggestion
        return make_response(jsonify({"message": f"No outstanding rentals for customer {customer.id} and video {video.id}"}), 400)
```